<?xml version="1.0" encoding="UTF-8"?>

<launch>
	<node name="yaml_position" pkg="play_motion" type="yaml_position" output="screen">  
        <rosparam command="load" file="$(find play_motion)/config/position_manipulation.yaml"/>
	<param name="node_path" type="string" value="$(find play_motion)" />
        </node> 
</launch>
